_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
```

Comment:
I wonder if this post should be removed entirely since we don't have an OmbuLabs shop anymore? We can create a separate story for that if you agree.

Reply:
I think we should keep it. It's a nice article. We can probably update it and remove the link to the shop

Reply:
@FionaDL could you submit a new story for this?